### PR TITLE
Propose 'Intake bridging' as the technique for remote -> matrix only

### PR DESCRIPTION
**Pull request by turt2live**
With a little shameless self-promotion, of course. Thanks TimePath for the recommendation for a name, and @t3chguy for the discussion.

Inserted just above sidecar bridging to avoid having to rewrite that entire section.

*Edited: I screwed up the attribution*

### CODE REVIEW

**Comment by richvdh** [comment]
@kegsay: I think you know something about bridges. can you sanity-check this?

**Comment by non-Jedi** [comment]
May want to mention that the twitter bridge can also function this way. Might be other, too, but I can't think of them off the top of my head.

File: supporting-docs/guides/2017-03-11-types-of-bridging.md [hunk 1]
```diff
@@ -62,6 +62,10 @@ Some remote protocols (IRC, XMPP, SIP, SMTP, NNTP, GnuSocial etc) support federa
 
 We’re not aware of anyone who’s done this yet.
 
+### Intake bridging
+
+Intake bridging is rare, but can be used to represent a bridge that is one-way from the remote system into matrix. This is common when the remote system does not permit message posting through approved channels, or is simply not capable of handling posting outside their system. The users bridged from the remote system often appear as virtual users in matrix, as is the case with [matrix-appservice-instagram](https://github.com/turt2live/matrix-appservice-instagram).
```
Comment (Kegsay) [inline supporting-docs/guides/2017-03-11-types-of-bridging.md]: > the remote system does not permit message posting through approved channels

Don't you mean:

> the remote system does not permit message posting through **unapproved** channels

**Comment by Kegsay** [comment]
Otherwise LGTM, though no-one will know what "Intake Bridging" means since it's a new term which is only referenced here. I'd prefer a more-descriptive term e.g "One-way bridging" which at least hints at what it might mean better than "Intake".

File: supporting-docs/guides/2017-03-11-types-of-bridging.md (hunk 1 above)
Comment (turt2live) [inline supporting-docs/guides/2017-03-11-types-of-bridging.md]: The intent is to say that the remote system doesn't have endpoints for posting messages. I'll update it accordingly (as both phrases are somewhat confusing, at least to me).

**Comment by turt2live** [comment]
@Kegsay Updated. Apologies for the delay.

**Comment by richvdh** [comment]
Thanks!

**Pull request merged**